src/test/scala/com/mozilla/telemetry/streaming/ExperimentEnrollmentsAggregatorTest.scala

```diff
@@ -0,0 +1,119 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+package com.mozilla.telemetry.streaming
+
+import java.sql.Timestamp
+import java.time.{LocalDateTime, ZoneOffset}
+
+import com.holdenkarau.spark.testing.DataFrameSuiteBase
+import org.apache.spark.sql.types._
+import org.apache.spark.sql.{DataFrame, SparkSession}
+import org.scalatest.{FlatSpec, GivenWhenThen, Matchers}
+
+class ExperimentEnrollmentsAggregatorTest extends FlatSpec with Matchers with GivenWhenThen with DataFrameSuiteBase {
+
+  val k = 10
+  val checkedColumns = Array("window_start", "window_end", "experiment_id", "branch_id", "enroll_count", "unenroll_count")
+  val ExpectedWindowStart = new Timestamp(LocalDateTime.parse("2016-04-07T13:35:00").toInstant(ZoneOffset.UTC).toEpochMilli)
+  val ExpectedWindowEnd = new Timestamp(LocalDateTime.parse("2016-04-07T13:40:00").toInstant(ZoneOffset.UTC).toEpochMilli)
+  private val ExperimentA = "pref-flip-timer-speed-up-60-1443940"
+  private val ExperimentB = "pref-flip-search-composition-57-release-1413565"
+
+  override def assertDataFrameEquals(expected: DataFrame, result: DataFrame): Unit = {
+    def prepare(df: DataFrame) =
+      df.select(checkedColumns.head, checkedColumns.tail: _*)
+        .orderBy("window_start", "experiment_id", "branch_id")
+
+    super.assertDataFrameEquals(prepare(expected), prepare(result))
+  }
+
+  def prepareExpectedAggregate(spark: SparkSession, rows: (String, String, Long, Long)*): DataFrame = {
+    import spark.implicits._
+    spark.sparkContext.parallelize(List[(Timestamp, Timestamp, String, String, Long, Long)](
```

Comment:
Since you're using `DataFrameSuiteBase` could `prepareExpectedAggregate` use the class SparkSession instead of having one passed in?